manifests/charts/istiod-remote/templates/configmap.yaml

```diff
@@ -3,16 +3,6 @@
     # Refer to https://github.com/spiffe/spiffe/blob/master/standards/SPIFFE-ID.md#21-trust-domain
     trustDomain: {{ .Values.global.trustDomain | quote }}
 
-    {{- if .Values.global.useMCP }}
```

Comment:
MCP is not used anymore. If they somehow need this, even though it does nothing, they can set through meshConfig directly

Reply:
Well, we added MCP-over-XDS - and technically only server-side MCP is gone, we still support client side. 

But that should be configured via mesh config, not global.